<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QMainWindow {
   color:black;
   background-color:black;
border:1px;
}
QLabel{
	color: #fff;
}
QLineEdit{
color: #fff;
}
#centralwidget{
	background-color:#1f232a;
}
#leftMenuSubContainer{
	background-color:#16191d;
   border-radius: 30px;
}
#bottomMenuSubContainer{
	background-color:#2c313c;
   border-radius: 30px;
}

QProgressBar{
    background-color:#2c313c;
	border-radius: 10px;

}
 QProgressBar::chunk {
     background-color:white;
	text-align:left;
	border-radius: 10px;
	
 }
QStackedWidget{
	background-color:#16191d;
	text-align:left;
	padding:5px 10px;
	border-radius: 25px;
}
QPlainTextEdit{
    background-color:#2c313c;
	border-radius: 30px;

}
QPushButton{
    background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
    color: #fff;
}
QPushButton:checked{
	background-color:#676e7b;
}
QPushButton:hover{
	background-color:#343b47;
}

QCheckBox{
color:#fff;
background-color:#fff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="centralWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="mainWindowContainer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QWidget" name="homePage">
             <property name="styleSheet">
              <string notr="true">QWidget{
background-color:#16191d

}
QPushButton{
    background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
    color: #fff;
}
QPushButton:checked{
	background-color:#676e7b;
}
QPushButton:hover{
	background-color:#343b47;
}
QPushButton::toolTip{
background-color:#676e7b;
}</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2"/>
            </widget>
            <widget class="QWidget" name="morePage">
             <property name="styleSheet">
              <string notr="true">QWidget{
background-color:#16191d

}</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6"/>
            </widget>
            <widget class="QWidget" name="mainPage">
             <property name="styleSheet">
              <string notr="true">QWidget{
background-color:#16191d

}
QLineEdit{
background-color:#2c313c;
border-radius: 10px;
}

QTextEdit{
background-color:#2c313c;
border-radius: 10px;
padding:5px 10px;
}

QPushButton{
    background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
}
QPushButton:checked{
	background-color:#676e7b;
}
QPushButton:hover{
	background-color:#343b47;
}

QComboBox{
background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
color:white;
}

QProgressBar{
    background-color:#2c313c;
	border-radius: 10px;

}
 QProgressBar::chunk {
     background-color:white;
	text-align:left;
	border-radius: 10px;
	
 }
QCheckBox::indicator{
border : 1px solid white;
border-radius: 2px;
}
QCheckBox::indicator::checked{
background-color:white;
}
QDoubleSpinBox{
	background-color:#2c313c;
	padding:5px 10px;
	border-radius: 10px;
	color:white;
}
QDoubleSpinBox::up-arrow {
       
		image: url(:/icons/icons/arrow-up.svg); 
    }
QDoubleSpinBox::down-arrow {

        image: url(:/icons/icons/arrow-down.svg); 
		
    }
QDoubleSpinBox::up-button {
        width: 20px;
        height: 15px;
    }
QDoubleSpinBox::down-button {
        width: 20px;
        height: 15px;
    }
</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QWidget" name="processContainer" native="true">
                <property name="maximumSize">
                 <size>
                  <width>16000</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*:disabled{
 
	color:gray;
}</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QScrollArea" name="scrollArea_2">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents_2">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>817</width>
                      <height>512</height>
                     </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                     <item row="0" column="0">
                      <widget class="QWidget" name="processSettingsContainer" native="true">
                       <layout class="QVBoxLayout" name="verticalLayout_17">
                        <property name="spacing">
                         <number>0</number>
                        </property>
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item>
                         <widget class="QWidget" name="inputFileContainer" native="true">
                          <layout class="QHBoxLayout" name="horizontalLayout_4">
                           <item>
                            <widget class="QLineEdit" name="inputFileText">
                             <property name="minimumSize">
                              <size>
                               <width>0</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="text">
                              <string/>
                             </property>
                             <property name="readOnly">
                              <bool>false</bool>
                             </property>
                             <property name="placeholderText">
                              <string>Paste YouTube link here</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QPushButton" name="getDataButton">
                             <property name="minimumSize">
                              <size>
                               <width>0</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="font">
                              <font>
                               <pointsize>15</pointsize>
                              </font>
                             </property>
                             <property name="styleSheet">
                              <string notr="true"/>
                             </property>
                             <property name="text">
                              <string>Get Data</string>
                             </property>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QWidget" name="outputFileContainer" native="true">
                          <layout class="QHBoxLayout" name="horizontalLayout_5">
                           <item>
                            <widget class="QPushButton" name="outputFileSelectButton">
                             <property name="enabled">
                              <bool>false</bool>
                             </property>
                             <property name="minimumSize">
                              <size>
                               <width>0</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="font">
                              <font>
                               <pointsize>15</pointsize>
                              </font>
                             </property>
                             <property name="styleSheet">
                              <string notr="true"/>
                             </property>
                             <property name="text">
                              <string>Select Output Folder</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QLineEdit" name="outputFileText">
                             <property name="enabled">
                              <bool>false</bool>
                             </property>
                             <property name="minimumSize">
                              <size>
                               <width>0</width>
                               <height>30</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QWidget" name="widget_3" native="true">
                          <layout class="QHBoxLayout" name="horizontalLayout_11">
                           <item>
                            <widget class="QLabel" name="label_12">
                             <property name="font">
                              <font>
                               <pointsize>15</pointsize>
                              </font>
                             </property>
                             <property name="text">
                              <string>Format</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QComboBox" name="mediaTypeComboBox">
                             <property name="sizePolicy">
                              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                               <horstretch>0</horstretch>
                               <verstretch>0</verstretch>
                              </sizepolicy>
                             </property>
                             <property name="minimumSize">
                              <size>
                               <width>0</width>
                               <height>0</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                             <item>
                              <property name="text">
                               <string>Video</string>
                              </property>
                             </item>
                             <item>
                              <property name="text">
                               <string>Audio</string>
                              </property>
                             </item>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QWidget" name="widget" native="true">
                          <layout class="QHBoxLayout" name="horizontalLayout">
                           <item>
                            <widget class="QLabel" name="label_7">
                             <property name="font">
                              <font>
                               <pointsize>15</pointsize>
                              </font>
                             </property>
                             <property name="text">
                              <string>Resolution</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QComboBox" name="resolutionComboBox">
                             <property name="sizePolicy">
                              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                               <horstretch>0</horstretch>
                               <verstretch>0</verstretch>
                              </sizepolicy>
                             </property>
                             <property name="minimumSize">
                              <size>
                               <width>250</width>
                               <height>0</height>
                              </size>
                             </property>
                             <property name="maximumSize">
                              <size>
                               <width>16777215</width>
                               <height>30</height>
                              </size>
                             </property>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>250</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_13"/>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="bottomMenuSubContainer" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">#bottomMenuSubContainer{
background-color:#1f232a
}
#progressBar{
background-color:#343b47
}
#startRenderButton{
background-color:#343b47
}
#startRenderButton:hover{
background-color:#676e7b
}</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="startDownloadButton">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>59</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset resource="resources.qrc">
                        <normaloff>:/icons/icons/arrow-down-circle.svg</normaloff>:/icons/icons/arrow-down-circle.svg</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>55</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QProgressBar" name="progressBar">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="value">
                       <number>0</number>
                      </property>
                      <property name="textVisible">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="settingsPage">
             <property name="styleSheet">
              <string notr="true">QWidget{
background-color:#16191d

}
QLineEdit{
background-color:#2c313c;
border-radius: 10px;
}

QTextEdit{
background-color:#2c313c;
border-radius: 10px;
padding:5px 10px;
}

QPushButton{
    background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
}
QPushButton:checked{
	background-color:#676e7b;
}
QPushButton:hover{
	background-color:#343b47;
}

QComboBox{
background-color:#2c313c;
	text-align:left;
	padding:5px 10px;
	border-radius: 10px;
color:white;
}

QProgressBar{
    background-color:#2c313c;
	border-radius: 10px;

}
 QProgressBar::chunk {
     background-color:white;
	text-align:left;
	border-radius: 10px;
	
 }
QCheckBox::indicator{
border : 1px solid white;
border-radius: 2px;
}
QCheckBox::indicator::checked{
background-color:white;
}

Line{
color:white;
background-color:white;
}
QDoubleSpinBox{
	background-color:#2c313c;
	padding:5px 10px;
	border-radius: 10px;
	color:white;
}
QDoubleSpinBox::up-arrow {
       
		image: url(:/icons/icons/arrow-up.svg); 
    }
QDoubleSpinBox::down-arrow {

        image: url(:/icons/icons/arrow-down.svg); 
		
    }
QDoubleSpinBox::up-button {
        width: 20px;
        height: 15px;
    }
QDoubleSpinBox::down-button {
        width: 20px;
        height: 15px;
    }
</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
